app/src/pages/documentSearchResultsPage/DocumentSearchResultsPage.tsx

```diff
@@ -34,8 +34,13 @@ function DocumentSearchResultsPage() {
     const baseUrl = useBaseAPIUrl();
     const baseHeaders = useBaseAPIHeaders();
     const config = useConfig();
-
     const mounted = useRef(false);
+
+    const resetDocState = () => {
+        setSubmissionState(SUBMISSION_STATE.INITIAL);
+        setDownloadState(SUBMISSION_STATE.INITIAL);
```

Comment:
Do we need to set SearchResults to an empty list here?
Also should it be initial state? not failed?